services/scoop/scoop.service.js

```diff
@@ -0,0 +1,95 @@
+'use strict'
+
```

Comment:
Let's go ahead and name the file `scoop-version.service.js` to match the classname and because there's a reasonably good chance we'll add other scoop-related badges in the future (for example licenses, etc.)

Reply:
Done in 97897c3f5001443a09e3677c8b660bbdd3daeab6.